lldp_8021qaz_cmds.c

```diff
@@ -1121,6 +1121,7 @@ static int get_arg_app(struct cmd *cmd, char *args, UNUSED char *arg_value,
 	const char *app;
 	u8 prio, sel;
 	int proto, hw = -1, i;
+	char *parse;
```

Comment:
Same comment here - initialize parse to `NULL` in case `for` loop never reached `strdup()`.